lib/node_modules/@stdlib/utils/get-prototype-of/lib/polyfill.js

```diff
@@ -22,6 +22,7 @@
 
 var nativeClass = require( '@stdlib/utils/native-class' );
 var getProto = require( './proto.js' );
+var Object = require( '@stdlib/object/ctor' );
```

Comment:
```suggestion
var Object = require( '@stdlib/object/ctor' );
var getProto = require( './proto.js' );
```

Incorrect require order.